backend/entities/application_entity.py

```diff
@@ -8,15 +8,20 @@
 from backend.entities.academics.section_entity import SectionEntity
 from backend.models.application_details import (
     ApplicationDetails,
-    New_UTADetails,
-    UTADetails,
+    NewUTAApplicationDetails,
+    UTAApplicationDetails,
 )
 from backend.models.academics.section import Section
 
 from .entity_base import EntityBase
 from typing import Self, Dict
-from ..models.application import Application, UTA, New_UTA, Returning_UTA
-from ..models.application_details import UTADetails
+from ..models.application import (
+    Application,
+    UTAApplication,
+    NewUTAApplication,
+    Returning_UTA,
```

Comment:
Nit: Might need to change the name of this too (although I know this is not really used yet)!